storage/range_tree.go

```diff
@@ -20,29 +20,286 @@ package storage
 import (
 	"github.com/cockroachdb/cockroach/proto"
 	"github.com/cockroachdb/cockroach/storage/engine"
+	"github.com/cockroachdb/cockroach/util"
 )
 
-// SetupRangeTree creates a new range tree.  This should only be called as part of
-// store.BootstrapRange.
-func SetupRangeTree(batch engine.Engine, ms *engine.MVCCStats,
-	timestamp proto.Timestamp, startKey proto.Key) error {
+type (
+	// RangeTree represents holds the metadata about the RangeTree.  There is
+	// only one for the whole cluster.
+	RangeTree proto.RangeTree
```

Comment:
Does it matter that these are methods on RangeTree instead of static functions that take a RangeTree as an argument? Type aliases like this can be awkward to work with so if you're not trying to implement an interface it's often simpler to stick with static functions.